questionnaires/forms.py

```diff
@@ -144,6 +151,9 @@ def clean(self):
                 if self.clean_errors:
                     form._errors = self.clean_errors
 
+        if should_be_multi_step:
+            self.add_error("multi_step", "Multi-step must be enabled with skip logic.")
```

Comment:
This is not the text that is specified in #1538.

Reply:
I have asked, in the issue, whether we ought to activate multi-step rather than warn the user.